### PR TITLE
Add link property 'ignore_if_unsupported'

### PR DESCRIPTION
**Pull request by mlsteele**
Add another field to sigchain v2 outer links. It's only used for team chains on the client.

### CODE REVIEW

**Comment by maxtaco** [comment]
shouldn't there be a check for inner/outer match?

**Comment by mlsteele** [comment]
There’s no inner component

**Comment by maxtaco** [comment]
but why not? it should also work for v1 links, no?

**Comment by mlsteele** [comment]
Added to inner

**Comment by mlsteele** [comment]
Would you take another look at this?

File: src/base.iced [hunk 1]
```diff
@@ -700,6 +715,9 @@ class OuterLink
 
     # For newer clients that push an explicit seq_type value, we push it onto the array here
     arr.push @seq_type if @seq_type?
+
```
Comment (maxtaco) [inline src/base.iced]: can you just update the comment about 3 lines above?

Comment (mlsteele) [reply src/base.iced]: isn't it still accurate?

Comment (maxtaco) [reply src/base.iced]:
```
# For backwards-compatibility, if the incoming chainlink doesn't have a
# seq_type, we don't push a null or a defaut value (3), we just leave it as
# a 5-value array
```

If you pass in a 6-value array, we don't add the ignore_if_unsupported for the same reason, is what I meant.

**Comment by mlsteele** [comment]
Ok, assuming that comment's good, ready for an npm push.

**Pull request merged**